mavros/src/lib/enum_to_string.cpp

```diff
@@ -213,6 +214,56 @@ std::string to_string(MAV_STATE e)
 
 	return mav_state_strings[idx];
 }
+
```

Comment:
@baumanta this is not right. The `to_string` function needs to be generated using `cog`. Check how the other enumerations are generated. You can also see with this add, you brake the checksum of line https://github.com/mavlink/mavros/pull/1118/files#diff-622b1c80b8e897ac3052c0d8b53ba2ddR267. You don't have to add this manually, you just need to let cog do it's job: https://github.com/mavlink/mavros/blob/master/CONTRIBUTING.md#cogpy-generators